src/Pact/Analyze/Types.hs

```diff
@@ -402,9 +367,12 @@ isConcreteS = isConcrete . _sSbv
 
 data QKind = QType | QAny
 
+-- Integer, Decimal, Bool, String, Time
+type SimpleType a = (Float a, Show a, SymWord a, SMTValue a)
```

Comment:
i like this

Reply:
ah that is great

Reply:
should `Float` definitely be in there too? or should we use a combination of Float + SimpleType without it?

Reply:
fixed -- thanks